### PR TITLE
[CPH-2018] add-cfp-and-location

### PR DESCRIPTION
**Pull request by JasmineCrozier**
The CFP is open and I added the venue

### CODE REVIEW

File: data/events/2018-copenhagen.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ startdate:  2018-04-24 # The start date of your event. Leave blank if you don't
 enddate:  2018-04-25 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
-cfp_date_announce:  # inform proposers of status
+cfp_date_start:  2018-01-10 # start accepting talk proposals.
+cfp_date_end:  2018-02-11 # close your call for proposals.
+cfp_date_announce:  open # inform proposers of status
```
Comment (kmugrage) [inline data/events/2018-copenhagen.yml]: This should be a date. It's meant to be when you'll let chosen speakers know you'd like them to come.

File: content/events/2018-copenhagen/location.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
 +++
 Title = "Location"
 Type = "event"
-Description = "Location for devopsdays Copenhagen 2018"
+Description = "Location for DevOps Days Copenhagen 2018"
```
Comment (kmugrage) [inline content/events/2018-copenhagen/location.md]: People choose CamelCase or lowercase often, but DevOpsDays is usually one word

File: data/events/2018-copenhagen.yml (hunk 1 above)
Comment (kmugrage) [reply data/events/2018-copenhagen.yml]: FYI, this is why the build is failing

File: content/events/2018-copenhagen/location.md (hunk 2 above)
Comment (bridgetkromhout) [reply content/events/2018-copenhagen/location.md]: But two words is okay too!

**Comment by JasmineCrozier** [comment]
@kmugrage @bridgetkromhout I think it's all fixed now. Thanks for your help.

**Comment by phrawzty** [comment]
Pas de problème concernant les petites fautes. Nous apprenons tous de nouvelles choses chaque jour ! 😄

**Pull request merged**